### ISSUE TITLE
Iteration Plan for release in May

### ISSUE THREAD

**Issue by SparkSnail** [microsoft/nni#2299]
Extending this short iteration to a regular iteration for April-May.
**Release Manager**: @sparksnail
**Code freeze date**:  5.19
**End Game Issue**: 5.25
**Target release date**: 5.26

---
### Code completed 
- [x] IT left work #2290 @chicm-ms 
### In progress

**General Supports Updates**
- [x] TF 2.0 https://github.com/microsoft/nni/issues/2127 @liuzhe-lz (3day + 2week)
   --- Using NNI NAS interface with keras/tf.keras #1737  
   --- Tensorflow pipeline 
   --- add example (ongoing, draft pr #2442 , 5.15)
- [x] 100W IPC limitation (ongoing , need to fix UT, #2425 @liuzhe-lz )
- [x] share code storage among trials in an experiements (@SparkSnail) #2314 (pr submitted, 5.15)
- [x] Show trial error message #2424  @chicm-ms @Lijiaoa (pr submitted, fixing comments,  5.11)
- [x] fix dev install @liuzhe-lz #2253 (PR #2372, need discussion )
- [x] Refactor overview top trials: @Lijiaoa  (pr submitted)
related issue: https://github.com/microsoft/nni/issues/2007  https://github.com/microsoft/nni/issues/2010
- [x] + "__version__" for SDK version @liuzhe-lz  (#2308) (ongoing, 5.15) 
Design issue: https://github.com/microsoft/nni/issues/2142
- [x] Webui feedback @Lijiaoa (1/2 pr submtted, #2433 , 5.15) 

**HPO Updates**
- [x] PBT left work and improvement @quanluzhang #2315 

**Feature Engineering Updates**
- [x] Feature engineering/selection (L1 ranker) TPOT (http://automl.info/tpot/) (@leckie-chn ) 

**NAS Updates**

- [x] improvement NAS API, add readability, improve user experience (@ultmaster ) #2316 
   ----Use OrderedDict for LayerChoice #2112 (merged)
   ----Fix bug (merged)
   ----Prettify the format of export #2389 (need fix, pr submitted, 5.15)
   ----Generate nni-free model after applying fixed architecture (pr submitted, 5.15)
   ----I'm not sure: combining sample_search and reset; combining sample_final and export(deferred)

**Model Compression Updates**
- [x] Model compression PyTorch 1.4 support, utility codes merge #2310 (@chicm-ms ) (pr submitted, fixing comments and bug)

**Training Service Updates**
- [x] support pai multiphase   @SparkSnail (pr submitted, fix comments)
- [x] update pai yaml merge #2309 @SparkSnail (pr submitted)

-----
## Bug Fix
open bugs: https://github.com/microsoft/nni/labels/bug
